src/tracing/writer.rs

```diff
@@ -221,11 +220,13 @@ impl<W: Write> TraceWriter<W> {
     }
 
     fn write_log(&mut self, log: &CallLog) -> io::Result<()> {
-        let log_style = self.log_style();
+        let log_name_style = self.log_name_style(log);
+        let log_param_style = self.log_param_style();
+
         self.write_branch()?;
 
         if let Some(name) = &log.decoded.name {
-            write!(self.writer, "emit {name}({log_style}")?;
+            write!(self.writer, "emit {log_name_style}{name}{log_name_style:#}({log_param_style}")?;
```

Comment:
@klkvr are you okay with rendering ansi escape codes in terminals that don't support colors? Otherwise I think it makes sense to keep moving forward with the `log.style` impl?